integration_tests/CMakeLists.txt

```diff
@@ -666,6 +666,7 @@ RUN(NAME bindpy_02           LABELS cpython c_py EXTRA_ARGS --link-numpy COPY_TO
 RUN(NAME bindpy_03           LABELS cpython c_py EXTRA_ARGS --link-numpy NOFAST COPY_TO_BIN bindpy_03_module.py)
 RUN(NAME bindpy_04           LABELS cpython c_py EXTRA_ARGS --link-numpy NOFAST COPY_TO_BIN bindpy_04_module.py)
 RUN(NAME bindpy_05           LABELS llvm_py c_py EXTRA_ARGS --enable-cpython COPY_TO_BIN bindpy_05_module.py REQ_PY_VER 3.10)
+RUN(NAME bindpy_06           LABELS llvm_py EXTRA_ARGS --enable-cpython COPY_TO_BIN bindpy_06_module.py REQ_PY_VER 3.10)
```

Comment:
You can add NOFAST to this. Also, I think we should be testing this with `cpython` as well to ensure what we support also works with CPython. We should not support something that is not supported with CPython.

Reply:
We should test bindpy_05.py with CPython.